src/main/java/hudson/tasks/Mailer.java

```diff
@@ -610,5 +652,6 @@ public UserProperty newInstance(StaplerRequest req, JSONObject formData) throws
     /**
      * Debug probe point to be activated by the scripting console.
      */
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "It's used for debugging")
```

Comment:
if some java plugin is setting the debug mode of the mailer it deserves to be broken IMHO.
If there is a legitimate use requiring this to be public then the justification and comment are incorrect.
I'll leave the decision on this to the maintainer.